pyproject.toml

```diff
@@ -1,3 +1,106 @@
 [build-system]
-requires = ["setuptools >= 64"]
+requires = ["setuptools>=61", "setuptools_scm"]
```

Comment:
Why reduce the version requirement for setuptools?

Why include `setuptools_scm`? According to its docs:

> setuptools-scm extracts Python package versions from git or hg metadata instead of declaring them as the version argument or in a Source Code Managed (SCM) managed file.

We hard-code the version in this file later on, so including this requirement serves no purpose (or does it? 🤷)

Reply:
turns out we actually need setuptools_scm for including package data